pkgs/os-specific/linux/aseq2json/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchFromGitHub, pkg-config, alsaLib, glib, json-glib }:
+
+let
+  repository = fetchFromGitHub {
```

Comment:
Removing that let in here and using sourceRoot makes the package more similar to other packages which saves headaches in the long run.

Reply:
That makes it a lot cleaner, thanks!